components/vf-dropdown/vf-dropdown.js

```diff
@@ -29,9 +31,16 @@ export function vfDropdown() {
         currentPos = null;
       }
     });
+    // to being able to prevent scrolling when user navigates with the keyboard
+    // this must be done in the keydown event
+    component.addEventListener("keydown", function(keyboardEvent) {
+      if (currentPos !== null && isNavigationKey(keyboardEvent)) {
+        keyboardEvent.preventDefault();
+      }
+    });
     component.addEventListener("keyup", function(keyboardEvent) {
-      // if user presses Enter key, open the dropdown
-      if (keyboardEvent.keyCode === 13) {
+      // if user presses Enter or Space key, open the dropdown
+      if (keyboardEvent.code === 'Enter' || keyboardEvent.code === 'Space') {
```

Comment:
@pacope92 I would suggest to use the keycodes for all key events, not sure how other language keyboards will work this code, so it is better to use keycodes instead of name.